addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -268,22 +268,22 @@ export class PosOrder extends Base {
         return this.lines.find((orderline) => orderline.skip_change) ? true : false;
     }
 
-    is_empty() {
+    isEmpty() {
         return this.lines.length === 0;
     }
 
     updateSavedQuantity() {
         this.lines.forEach((line) => line.updateSavedQuantity());
     }
 
-    assert_editable() {
+    assetEditable() {
```

Comment:
@sjai-odoo Can you please correct this ?

Reply:
Hi @vlst-odoo 
Thanks, updated [in this pr](https://github.com/odoo/odoo/pull/189565)